### ISSUE TITLE
ISSUES WITH LOGR INFINITE VALUE

### ISSUE THREAD

**Issue by Mariamamghar1** [broadinstitute/ichorCNA#126]
I am running ichorcna as a tool to analyze my cfDNA data WGS. I do have an issue. One of the further analysis I would like to perform is to run GISTIC to get a bit more information on the data, and I need the logR copy number value, which in some of my samples appears as infinite. Could you support me with the? Do you have an idea of which value I could replace instead? 
[sample with inf.txt](https://github.com/broadinstitute/ichorCNA/files/12522747/sample.with.inf.txt)

**Comment by brmprnk** [comment]
Hey there, have you found a solution to your problem already? I'm running into the same issue and cannot determine the reason for the infinite values. Thanks!

**Comment by Mariamamghar1** [comment]
> Hey there, have you found a solution to your problem already? I'm running into the same issue and cannot determine the reason for the infinite values. Thanks!

No, i did not find a solution and no one replied to my question. still stuck with it
